Changelog.md

```diff
@@ -17,6 +17,7 @@ Bugfixes:
  * Control Flow Graph: Perform proper virtual lookup for modifiers for uninitialized variable and unreachable code analysis.
  * Immutables: Fix wrong error when the constructor of a base contract uses ``return`` and the parent contract contains immutable variables.
  * IR Generator: Fix IR syntax error when copying storage arrays of structs containing functions.
+ * IR Generator: Fix IR syntax error when copying reference types in calldata and storage to struct members in memory.
```

Comment:
```suggestion
 * IR Generator: Fix IR syntax error when copying reference types in calldata and storage to struct or array members in memory.
```
@ekpyron would that work? I can commit and squash it afterwards.

Reply:
Hmm. Github is not rendering it properly "<span class="x x-first x-last">or array </span>members"!